task/sql-tasks.js

```diff
@@ -177,18 +273,32 @@ async function task_1_12(db) {
  *
  */
 async function task_1_13(db) {
-    throw new Error("Not implemented");
+    let result = await db.query(`
+        SELECT

```

Comment:
this is good. Try also to write it like this

```
select
   (select count()...) as 'T'
   (select count()...) as 'TT'

```

notice from clause is optional